flocx_market/api/offer_contract_relationship.py

```diff
@@ -9,9 +9,11 @@ class OfferContractRelationship(Resource):
     def get(cls, marketplace_offer_id=None, contract_id=None):
         if (contract_id is None) and (marketplace_offer_id is None):
             return [x.to_dict() for x in
-                    ocr.OfferContractRelationship.get_all()]
-        ocrs = ocr.OfferContractRelationship.get(marketplace_offer_id,
-                                                 contract_id)
+                    ocr.OfferContractRelationship.get_all(g.context)]
+        ocrs = ocr.OfferContractRelationship.get(
+            marketplace_offer_id=marketplace_offer_id,
+            contract_id=contract_id,
+            context=g.context)
```

Comment:
The context should not be optional. I'd put it as the first argument, like:

```
        ocrs = ocr.OfferContractRelationship.get(g.context,
            marketplace_offer_id=marketplace_offer_id,
            contract_id=contract_id)
```

Reply:
The context is not optional? I was just passing in the arguments out of order, as some of the calls to the offercontractrelationsip object were out of order. But in the method signature it is not optional. Or am I misunderstanding something

Reply:
the signature in the api is
def offer_contract_relationship_get(context,
                                    marketplace_offer_id=None,
                                    contract_id=None):